features/include/pcl/features/spin_image.h

```diff
@@ -131,7 +133,25 @@ namespace pcl
       void 
       setImageWidth (unsigned int bin_count)
       {
-        image_width_ = bin_count;
+        const unsigned int necessary_desc_size = (bin_count+1)*(2*bin_count+1);
+        if (necessary_desc_size > static_cast<unsigned int>(PointOutT::descriptorSize())) {
+          for(int i=0; ; ++i) // Find the biggest possible image_width_
```

Comment:
Could we add some brackets for readability?

Reply:
Sure, done